src/core/server/http/router/request.ts

```diff
@@ -199,10 +205,12 @@ export class KibanaRequest<
     private readonly withoutSecretHeaders: boolean
   ) {
     // The `requestId` and `requestUuid` properties will not be populated for requests that are 'faked' by internal systems that leverage
-    // KibanaRequest in conjunction with scoped Elaticcsearch and SavedObjectsClient in order to pass credentials.
+    // KibanaRequest in conjunction with scoped Elasticsearch and SavedObjectsClient in order to pass credentials.
     // In these cases, the ids default to a newly generated UUID.
-    this.id = (request.app as KibanaRequestState | undefined)?.requestId ?? uuid.v4();
-    this.uuid = (request.app as KibanaRequestState | undefined)?.requestUuid ?? uuid.v4();
+    const appState = request.app as KibanaRequestState | undefined;
+    this.id = appState?.requestId ?? uuid.v4();
+    this.uuid = appState?.requestUuid ?? uuid.v4();
+    this.rewrittenUrl = appState?.rewrittenUrl;
```

Comment:
Should we default to `request.url` or do we want `rewrittenUrl` to be explicitly undefined when no interceptor rewrites it? Either way, we probably should add a test for this behavior.

Reply:
I'd prefer it to be `undefined`. I will add a test